app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<h1 class='title'> Recipe 2</h1>
+<div class='recipe-detail-container'>
+ <div class='preparation'>
+   <p class='preparation-info'>Preparation time: <%= @recipe.preparation_time.strftime('%H:%M') %> hours </p>
+   <p class='preparation-info'>Cooking time: <%= @recipe.cooking_time.strftime('%H:%M') %>hours</p>
+   <p class='preparation-info'>Steps go here...</p>
+   <button class='add-btn'>Generate shopping list</button> 
+ </div>
+ <div class='public-info'>
+ <div class='public-container'>
+ <div >
+  <p class='public-title'>
+  Public
+  </p>
+  </div>
+  <div class='public-form'>
+   <%= form_with(model: @recipe, url: toggle_public_recipe_path(@recipe), method: :patch, local: true) do |form| %>
+    <button type="submit" class="icon-button">
+      <% if @recipe.public? %>
+        <%= icon('fas', 'toggle-on', class: 'fa-2x') %>
+      <% else %>
+        <%= icon('fas', 'toggle-off', class: 'fa-2x') %>
+      <% end %>
+    </button>
+  <% end %>
+  </div>
+  
+  </div>
+
+  
+  <% if can?(:create, @recipe) %>
+  <%= link_to new_food_path do %>
+    <button class='add-btn'>Add food</button>
+  <% end %>
+ <% end %>  
+ </div>
+</div>
+
+<table>
+ <tr>
+ <th>Food</th>
+ <th>Measurement Unit</th>
+ <th>Value</th>
+ <th>Actions</th>
+ </tr>
+ <% @recipe_foods.each do |recipe_food| %>
+ <tr>
+ <td><%= recipe_food.food.name if recipe_food.food %></td>
+ <td><%= recipe_food.food.quantity if recipe_food.food %> </td>
+ <td><%= recipe_food.food.price if recipe_food.food %></td>
+ <td>
+  <button>Modify</button>
+  <button>Remove</button>
+ </td>
+ </tr>
+ <% end %>
+ <tr>
+</table>
```

Comment:
![image](https://github.com/ismayito/Rails-recipe-app/assets/59397159/26b7581f-1edc-43fc-90d6-e863cd29d622)


---

- In the recipe show page, I don't see any by this wireframe. Could you double-check this?

Reply:
Likewise to this here is what Ihave when I run the app;
![Screenshot from 2024-01-20 08-24-34](https://github.com/ismayito/Rails-recipe-app/assets/47735113/018215ee-0b5b-4f02-8b58-f42852935de5)

Reply:
- [  ] As the previous reviewer mentioned, the page does not exactly match the given Wireframe. For instance, there is no `Add Ingredient` button, hence user cannot add ingredients to the table thereby making the table blank. Kindly fix this

Your design:   👇 
![rails recipe](https://github.com/ismayito/Rails-recipe-app/assets/66097371/de907120-800a-4a42-917d-737d00b53970)

Reply:
Fixed. design matched the Given Wireframe 
![image](https://github.com/ismayito/Rails-recipe-app/assets/93598189/d188697c-980a-48e3-88f3-fc211ff1dbbc)

Reply:
- [ Optional ] Kindly I suggest incorporating a navigation bar on all recipe pages for enhanced user interaction. Including a navigation bar will facilitate seamless redirection to other pages, providing users with a more engaging and interactive experience.

![image](https://github.com/ismayito/Rails-recipe-app/assets/93598189/ab6b8674-7fe0-4eb6-814e-6a64313fab4d)